Documentatie/EP-online-validaties.md

```diff
@@ -36,33 +37,34 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
 
 | Technische naam 																| Rule(s)
 |-------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-|  CheckBagBerthId																|	BagResidenceId, BagBuildingId en BagPitchId mogen niet ingevuld zijn wanneer BagBerthId ingevuld is.
-|  CheckBagBuildingIdAndOrResidenceId											|	BagPitchId en BagBerthId mogen niet ingevuld zijn wanneer BagResidenceId en/of BagBuildingId ingevuld is/zijn.
-|  CheckBagPitchId																|	BagResidenceId, BagBuildingId en BagBerthId mogen niet ingevuld zijn wanneer BagPitchId ingevuld is.	
+|  CheckBagBuildingId															|	Het is verplicht een BAG Verblijfsobject Id op te geven wanneer er een Pand Id is opgegeven.
```

Comment:
Naamgeving gelijk trekken, ook in de 3 hieronder.
Nederlandse term gebruiken (evt met engelse tussen haakjes, zie verderop in file) en bag kan weggelaten worden.